src/modules/dashboard/components/ActionsPage/StakingValidationError/StakingValidationError.tsx

```diff
@@ -40,11 +42,31 @@ const stakeValidationMSG = defineMessage({
 
 const displayName = 'StakingValidationError';
 
-const StakingValidationError = ({ stakeType, errorValues }: Props) => (
-  <div className={styles.validationError}>
-    <FormattedMessage {...stakeValidationMSG[stakeType]} values={errorValues} />
-  </div>
-);
+const StakingValidationError = ({ stakeType, errorValues }: Props) => {
+  const { setIsOpen: openTokenActivationPopover } = useContext(
+    TokenActivationContext,
+  );
+
+  if (stakeType === 'tokens') {
+    return (
+      <Button
+        text={stakeValidationMSG[stakeType]}
```

Comment:
As the stakeType will always be 'tokens', theres no need to perform a lookup for the MSG.

Reply:
fixed